docs/whatsnew/1.1.rst

```diff
@@ -0,0 +1,182 @@
+.. doctest-skip-all
+
+.. _whatsnew-1.1:
+
+************************
+What's New in SunPy 1.1?
+************************
+
+Overview
+========
+
+The SunPy project is pleased to announce the 1.1 release of the sunpy package.
+
+The headline changes in 1.1 are:
+
+* The `~sunpy.coordinates` subpackage now supports four additional coordinate frames (HCI, HEE, GSE, and GEI).
+* Remote Data Manager (#3124)
+* SUVI support (#3269 / #3301)
```

Comment:
Spell out acronym or add link to somewhere else? (I have no idea what SUVI is...)

Reply:
I copy pasted these bullet points so I knew what I was putting in the doc, they should all probably be re-written like Albert did with the first one. :smile: